db/appointments.ts

```diff
@@ -1,12 +1,14 @@
 import { Appointment, AppointmentWithPatient } from '../types/Appointment';
 import date from '../utils/date';
-import { appointmentsTable, patientsTable } from './tables';
+import {
+  appointmentsTable, patientsTable, psychologistsTable, universitiesTable,
+} from './tables';
 import db from './db';
 
 const getAll = async (psychologistId: string): Promise<AppointmentWithPatient[]> => {
   try {
     const appointmentArray = await db.from(patientsTable)
-    .innerJoin(`${appointmentsTable}`, `${patientsTable}.id`, `${appointmentsTable}.patientId`)
+    .innerJoin(appointmentsTable, `${patientsTable}.id`, `${appointmentsTable}.patientId`)
     .where(`${appointmentsTable}.psychologistId`, psychologistId)
     .whereNot(`${appointmentsTable}.deleted`, true)
```

Comment:
Not in this PR but I find it hard to read.. it will be easier `.where(`${appointmentsTable}.deleted`, false)`